collections/src/main/scala/strawman/collection/mutable/Builder.scala

```diff
@@ -45,6 +45,9 @@ trait Builder[-A, +To] extends Growable[A] { self =>
     }
   }
 
+  @deprecated("Use sizeHint(coll, 0) instead of sizeHint(coll)", "2.13.0")
+  @`inline` final def sizeHint(coll: strawman.collection.Iterable[_]): Unit = sizeHint(coll, 0)
```

Comment:
What do you think of simply having a default value `0` for the `delta` parameter?

Reply:
Makes sense